app/controllers/state_file/questions/waiting_to_load_data_controller.rb

```diff
@@ -7,6 +7,10 @@ class WaitingToLoadDataController < QuestionsController
       def edit
         raise ActionController::RoutingError, 'Not Found' unless params[:authorizationCode]
         return redirect_to next_path if current_intake.raw_direct_file_data.present?
+        if [nil, "unfilled"].include?(current_intake.consented_to_terms_and_conditions)
+          flash[:alert] = I18n.t("general.one_intake_at_a_time")
+          redirect_to StateFile::StateFilePagesController.to_path_helper(action: :login_options, us_state: current_intake.state_code)
+        end
```

Comment:
this should ship separately (and first, ideally). can you extract this into its own pr?